android/assets/jsons/translations/template.properties

```diff
@@ -1187,6 +1187,12 @@ Number of your cities celebrating\n'We Love The King Day' thanks\nto access to t
 WLTK demand = 
 WLTK- = 
 Number of your cities\ndemanding this resource for\n'We Love The King Day' = 
+Social Policies: = 
+At war with [enemy] = 
+Friends with [civName] = 
+( [numberOfTurns] Turns Left) = 
```

Comment:
I feel like this entry shouldn't be in parentheses. Maybe do something like `( {[numberOfTurns] Turns Left})` in code then have `[numberOfTurns] Turns Left)` as the entry. The braces will handle the translation without needing translators to observe the parentheses and any extra spacing.

Reply:
> `[numberOfTurns] Turns Left)`

I assume you meant without the closing bracket right?

Reply:
Yes, that was a type

Reply:
I updated the picture. Thanks for the suggestions :D

Reply:
Oh nice why haven't I thought about that :)) thanks!